python/fnllm/fnllm/utils/sliding_window.py

```diff
@@ -1,8 +1,8 @@
-# Copyright (c) 2024 Microsoft Corporation.
+# Copyright (c) 2025 Microsoft Corporation.
 
 """Implementation of a sliding window based on time to calculate a moving average."""
 
-# Copyright (c) 2024 Microsoft Corporation.
+# Copyright (c) 2025 Microsoft Corporation.
```

Comment:
Is this a duplicate copyright line? should we remove this one?

Reply:
duplicate, good catch!